src/v2i-hub/TimPlugin/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+	"name":"TIM",
+	"description":"Provides Traveller Information Message (TIM).",
+	"version":"3.2.0",
+	"exeLocation":"/bin/TimPlugin",
+	"coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+	"messageTypes":[
+		{
+			"type":"J2735",
+			"subtype":"TIM",
+			"description":"Travelor Information Message indicating the active Work Zone information."
+		}
+	],
+	"configuration":[
+		{
+			"key":"Frequency",
+			"default":"1000",
+			"description":"The frequency to send the TIM in milliseconds."
+		},
+		{
+			"key":"MapFile",
+			"default":"IVP_GF_TIM.xml",
+			"description":""
+		},
+		{
+			"key":"Snap Interval",
```

Comment:
Is this still applicable to TIM plugin - why would TIM plugin track BSMs?

Reply:
Addressed